package.json

```diff
@@ -8,14 +8,27 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "start": "node index.js",
+    "start": "node ./src",
     "develop": "nodemon yarn start",
     "lint": "eslint .",
-    "test": "jest"
+    "test": "jest",
+    "test:watch": "jest --watch"
```

Comment:
Not sure about this. Having `yarn test:watch` is very close in length to `yarn test --watch`; perhaps we could consider instead adding the `--watch` command to the `test` alias so it always goes into watch mode when we run `yarn test`?